src/proxy-page/steps/addJira.ts

```diff
@@ -211,135 +238,178 @@ export default (config: ConfigService, jiraService: JiraService): Step => async
       // Load new base URL if defined a specific connection for Jira as ENV variables
       // otherwise default to standard baseURL defined for main server
       const baseUrl = process.env[`CPV_JIRA_${server.replace(/\s/, '_')}_BASE_URL`]
-          ?? config.get('confluence.baseURL');
+        ?? config.get('confluence.baseURL');
       issues.forEach((issue) => {
         data.push({
-          key: {
-            name: issue.key,
-            link: `${baseUrl}/browse/${issue.key}?src=confmacro`,
-          },
+          reporter: issue.fields.reporter?.displayName || '',
+          components: getContentWithMap(issue.fields.components),
+          acceptance_criteria: getTextContent(issue.fields.customfield_10042),
+          detail_design_reference: getTextContent(issue.fields.customfield_10129),
+          storypoints: issue.fields.customfield_10026 || '',
+          labels: issue.fields.labels || [],
+          sprint: getContentWithMap(issue.fields.customfield_10020),
+          key: createLinkObject(issue.key, baseUrl),
+          parent: createLinkObject(issue.fields.parent?.key, baseUrl),
+          subtasks: getSubtasksInfo(issue.fields.subtasks, baseUrl),
+          issuelinks: getIssueLinksInfo(issue.fields.issuelinks, baseUrl),
           t: {
-            name: issue.fields.issuetype.name,
-            icon: issue.fields.issuetype?.iconUrl,
+            name: issue.fields.issuetype.name || '',
+            icon: issue.fields.issuetype?.iconUrl || '',
           },
-          summary: {
-            name: issue.fields.summary,
-            link: `${baseUrl}/browse/${issue.key}?src=confmacro`,
-          },
-          updated: issue.fields.updated
-            ? `${new Date(issue.fields.updated).toLocaleString('en-EN', {
-              year: 'numeric',
-              month: 'short',
-              day: '2-digit',
-              hour: '2-digit',
-              minute: '2-digit',
-            })}`
-            : '',
-          assignee: issue.fields.assignee?.displayName,
+          summary: createLinkObject(issue.key, baseUrl, issue.fields.summary),
+          updated: formatDate(issue.fields.updated),
+          startdate: formatDate(issue.fields.customfield_10015),
```

Comment:
This `customfield_10015` looks like a custom field for us.
What would happen if we run this code in another Jira/Confluence instance?